examples/tensorflow/translation/run_translation.py

```diff
@@ -339,7 +339,7 @@ def main():
             use_auth_token=True if model_args.use_auth_token else None,
         )
     # See more about loading any type of standard or custom dataset (from files, python dict, pandas DataFrame, etc) at
-    # https://huggingface.co/docs/datasets/loading_datasets.html.
+    # https://huggingface.co/docs/datasets/main/loading
```

Comment:
Let's point to the official release version, rather than on main, as that's what most people will be working from

```suggestion
    # https://huggingface.co/docs/datasets/loading
```